src/test/php/xml/unittest/NodeTest.class.php

```diff
@@ -194,4 +194,19 @@ public function fromObject() {
       ]), 'node'))
     );
   }
+
+  #[@test]
+  public function fromObjectShortName() {
+
+    $obj= new Object([
+        '__construct' => function() {}
+      ]);
+
```

Comment:
* Your identing style seems random.
* Your use of whitespace seems random.
* Object does not accept any constructor parameters (you are passing a hashmap with an empty closure to the constructor, not defining a constructor like with `newinstance`).